### PR TITLE
Conflict with signinfo's temporary linguistic node sequence file caused test failures. 

### PR DESCRIPTION
**Pull request by gknisely**
# Issue

graphparser test is randomly failing.   Ref: https://github.com/valhalla/valhalla/pull/4606#issuecomment-1977634974

## Tasklist

 - [ ] Add tests
 - [ ] Add #fixes with the issue number that this PR addresses
 - [ ] Update the docs with any new request parameters or changes to behavior described
 - [x] Update the [changelog](CHANGELOG.md)
 - [ ] If you made changes to the lua files, update the [taginfo](taginfo.json) too.

## Requirements / Relations

 Link any requirements here. Other pull requests this PR is based on?

### CODE REVIEW

File: test/graphparser.cc [hunk 1]
```diff
@@ -677,7 +678,7 @@ void DoConfig() {
     file << "{ \
       \"mjolnir\": { \
       \"id_table_size\": 1000, \
-      \"tile_dir\": \"test/data/parser_tiles\" \
```
Comment (kevinkreiser) [inline test/graphparser.cc:680]: utrecht was using this? at this point its happened to us so many times that we had directory collisions we might try having the code generate a random suffix for the directories we create :smile:

**Review by kevinkreiser — APPROVED** [review]

File: test/graphparser.cc (hunk 1 above)
Comment (gknisely) [reply test/graphparser.cc:680]: yes.  https://github.com/valhalla/valhalla/blob/master/test/utrecht.cc#L55

Comment (gknisely) [reply test/graphparser.cc:680]: multiple locations

**Comment by gknisely** [comment]
Still having issues.  Looking at it

File: test/graphparser.cc (hunk 1 above)
Comment (nilsnolde) [reply test/graphparser.cc:680]: > we might try having the code generate a random suffix for the directories we create

I’d be in favor of that. As long as we print it out it’d be a good time investment. Would fit well in your other test cleanup PR:)

Comment (gknisely) [reply test/graphparser.cc:680]: Determined that this was not the cause of the failures

File: test/signinfo.cc [hunk 2]
```diff
@@ -30,7 +30,7 @@ TEST(Signinfo, ExitToTest) {
   std::vector<SignInfo> signs;
   std::vector<std::string> linguistics;
   std::vector<std::pair<std::string, bool>> default_languages;
-  const std::string linguistic_node_file = "test_linguistic_node.bin";
+  const std::string linguistic_node_file = "test_sign_linguistic_node.bin";
```
Comment (gknisely) [inline test/signinfo.cc:33]: This file was the cause of the problem.  In the multi-threaded environment the `test_linguistic_node.bin` was getting deleted for the `graphparser.cc` test.

**Review by dnesbitt61 — DISMISSED** [review]

**Review by dnesbitt61 — APPROVED** [review]

File: test/signinfo.cc (hunk 2 above)
Comment (nilsnolde) [reply test/signinfo.cc:33]: so this test also builds to the same directory as `graphparser.cc`? if so, shouldn't we change that?

Comment (gknisely) [reply test/signinfo.cc:33]: Most are all running in the same directory.  This test is testing specifically signs and is not building data.  This is a temporary file as it is required for `CreateSignInfoList`  We are not testing linguistic node signage here as it is done in the gurka tests.  The real solution is to migrate the old tests to gurka.

**Pull request merged**